packages/browser/src/types.ts

```diff
@@ -474,12 +474,19 @@ export interface PostHogConfig {
     disable_cookie?: boolean
 
     /**
-     * Determines whether PostHog should disable surveys.
+     * Determines whether PostHog should disable all surveys functionality.
      *
      * @default false
      */
     disable_surveys: boolean
 
+    /**
+     * Determines whether PostHog should disable automatic display of surveys. If this is true, popup or widget surveys will not be shown when display conditions are met.
+     *
+     * @default false
+     */
+    disable_surveys_automatic_display: boolean
```

Comment:
It's already merged, but the name and description are confusing to me.
we have `disable_surveys` and `disable_surveys_automatic_display`
why would I chose `disable_surveys_automatic_display` over `disable_surveys`?
They do the same thing, I guess? I probably lack some context, and the user probably will too